specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+
+This card is another possible example of a use case for non-adjacent labels, where there is additional information between the label and the input:
+
+![img](assets/InputLabels/AdditionalLabelContent.png)
+
+This scenario could be covered by a RichTextBlock that contains both pieces of text being used as the label (arguably the entire text should be read as the accessibility label for the input). 
+
+This approach will limit what elements can be rendered as the label to only text.
+
+## Backwards Compatibility
+On earlier versions of renderers the label would be completely dropped, leading to a bad user experience. The card may be completely unusable if without the labels the user cannot deduce what information is meant to be provided in the unlabeled inputs.
+
+Because the label is a property, not a new type, card authors would not be able to use type to trigger fallback to an independent text block and input. To support this, the card author would need to use requires as follows:
+
+```json
+{
+	"type": "Input.Text",
+	"id": "NameId",
+	"label": "Please enter your name",
+	"requires": {
+		"adaptiveCards": "1.3"
+	},
+	"fallback": {
+		"type": "Container",
+		"items": [
+			{
+				"type": "TextBlock",
+				"text": "Please enter your name:"
+			},
+			{
+				"type": "Input.Text",
+				"id": "NameId"
+			}
+		]
+    }
+}
+```
+We could also possibly add a specific named feature to the requires dictionary to allow card authors to address it directly using something like:
+```json
+"requires": {
+	"inputLabels": "1.0"
+}
+```
+Either of these approaches would allow card authors who know their cards will be hitting platforms targeting both 1.2 and 1.3 to take advantage of the labels for accessibility or interactivity purposes in 1.3 without breaking their experience on 1.2 clients. That this will not help card authors who are targeting 1.0 or 1.1 hosts where fallback is not supported. Card authors who know their hosts support version 1.3 (or any future version) will not need to worry about this concern.
+
+## Placeholders
+
+Based on the answers from Q7, no users answered to using placeholders only for labeling their inputs. For v1 of this feature we decided to not modify placeholders.
+
+## Label Position
+
+As alluded to above, we should consider how the label position is determined. One of the goals of this feature is to allow us to someday intelligently reflow inputs, so it may be the case that we want to maintain control of label positioning. Today we would always draw the label above the input, but in the future we may put it to the side if our rendering space is wide enough, or push it into placeholders if our space is constrained. For the time being, we can consider label position to be work for the future so it can line with intelligent reflowing so on v1 it's not going to be considered. 
```

Comment:
Great - let's annotate this P2 and also mention in teh proposal upfront that labels will be positioned on top of the input. @siduppal will this work for you guys?